### PR TITLE
html-tables - Susana Gutierrez

### PR DESCRIPTION
**Pull request by susieggrim**
html-tables

### CODE REVIEW

File: html-tables/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      <thead>
+        <tr>
+          <th>Student Name</th>
+          <th>Course</th>
+          <th>Grade</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <th>Grace Hopper</th>
```
Comment (BrettAlbright) [inline html-tables/index.html]: the issue with the centering is because you are using `th` elements instead of `td` elements

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
small update will fix the display!

**Comment by susieggrim** [comment]
html-tables-2

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**